src/utils/__tests__/ReactChildren-test.js

```diff
@@ -94,7 +112,8 @@ describe('ReactChildren', function() {
       return this;
     };
 
-    var scopeTester = {};
+    // TODO: Use an object to test, after non-object fragments has fully landed.
```

Comment:
I use the ReactChildren helpers to iterate over the result of a helper. Since an object is not terminal, yet, I can't use that to test. Once we fix this fragment thing, we can revert to just use the object here.